src/main/resources/application.properties

```diff
@@ -1,8 +1,7 @@
-# botkit
-spring.jackson.property-naming-strategy=SNAKE_CASE
-
-# slack integrations
-rtmUrl=https://somewhere.com
-slackBotToken=abcdefg
-slashCommandToken=abcdefg
-slackIncomingWebhookUrl=https://somewhere-else.com
+rtmUrl=
+slackBotToken=${SLACK_BOT_TOKEN}
```

Comment:
The environment the app is running in on Heroku. There's a web interface I use to configure these.